### PR TITLE
Remove unexpected macros calls

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

these macroses should not be called here

and, if it is intended to show the `style()` usage is experimental, per bcd, it should not be considered as experimental, too

### CODE REVIEW

File: files/en-us/web/css/@container/index.md [hunk 1]
```diff
@@ -260,8 +260,6 @@ The following query evaluates to true and applies the declared style if the cont
 
 ### Container style queries
 
-{{CSSRef}}{{SeeCompatTable}}
```
Comment (estelle) [inline files/en-us/web/css/@container/index.md:263]: @container queries are not experimental, but the `style()` value is not yet fully supported. While this macro should be removed, we should add that, unlike the `@container` in general, `style()` is not Baseline

Comment (skyclouds2001) [reply files/en-us/web/css/@container/index.md:263]: bcd consider it as non-experimental

image

aand currently only Firefox does not support this feature

see also https://github.com/mdn/browser-compat-data/blob/main/css/at-rules/container.json

Comment (skyclouds2001) [reply files/en-us/web/css/@container/index.md:263]: bcd consider it as non-experimental

image

aand currently only Firefox does not support this feature

see also https://github.com/mdn/browser-compat-data/blob/main/css/at-rules/container.json

Comment (estelle) [reply files/en-us/web/css/@container/index.md:263]: experimental should definitely be removed, but we need to add something in its place.

`@container` is generally Baseline, but `style()`  container queries are not baseline. Yes, you are right to remove the "experimental" from the `style()` section, as this PR does. My question is,  how should we point out that while the feature is generally Baseline, part of this feature is not? we need to add something when we remove the warning. I am not sure what that something is though.

File: files/en-us/web/css/@container/index.md [hunk 2]
```diff
@@ -260,8 +260,6 @@ The following query evaluates to true and applies the declared style if the cont
...
-
 Container queries can also evaluate the computed style of the container element. A _container style query_ is a `@container` query that uses one or more `style()` functional notations. The boolean syntax and logic combining style features into a style query is the same as for [CSS feature queries](/en-US/docs/Web/CSS/CSS_conditional_rules/Using_feature_queries).
```
Comment (OnkarRuikar) [inline files/en-us/web/css/@container/index.md:263]: As per BCD rules, it's not experimental anymore, so to point out that Firefox doesn't support it, we could simply add a note about it.

```suggestion

> [!NOTE]
> The Firefox browser does not support container `style()` queries yet.

Container queries can also evaluate the computed style of the container element. A _container style query_ is a `@container` query that uses one or more `style()` functional notations. The boolean syntax and logic combining style features into a style query are the same as for [CSS feature queries](/en-US/docs/Web/CSS/CSS_conditional_rules/Using_feature_queries).
```

**Review by estelle — APPROVED** [review]
the same issue exists on other pages, so i guess no edit is needed.

thanks.

**Pull request merged**